### PR TITLE
Show all variables with the project specific values in the document variables tab

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
* **#2866** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2866?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @nickgraz and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/AcceleratorRouter/Documents/DocumentView/DocumentVariablesTab.tsx [hunk 1]
```diff
@@ -162,10 +161,12 @@ const DocumentVariablesTab = ({ document: doc, setSelectedTab }: DocumentVariabl
   );
 
   const onUpdate = () => {
-    dispatch(requestListVariables(doc.variableManifestId));
+    dispatch(requestListAllVariables());
     dispatch(requestListVariablesValues({ projectId: doc.projectId }));
   };
 
+  const onSearch = (str: string) => setSearchValue(str);
```
Comment (nickgraz) [inline src/scenes/AcceleratorRouter/Documents/DocumentView/DocumentVariablesTab.tsx:168]: I noticed that this does not actually do anything, the search value isn't checked anywhere. I will fix in a subsequent PR

Comment (nickgraz) [reply src/scenes/AcceleratorRouter/Documents/DocumentView/DocumentVariablesTab.tsx:168]: https://github.com/terraware/terraware-web/pull/2867

**Review by joshbuchea — APPROVED** [review]

**Comment by nickgraz** [comment]
### Merge activity

* **Jul 17, 5:39 PM EDT**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2866).

**Pull request merged**